projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -91,7 +92,28 @@ const LineChart: FC< LineChartProps > = ( {
 
 	const theme = buildChartTheme( chartTheme );
 
-	//
+	const handlePointerMove = useCallback( ( params: EventHandlerParams< DataPointDate > ) => {
```

Comment:
This looks more complex than the previous example. XYChart has its own tooltip, so maybe we could leverage it. Do you know if the docs suggest any benefit of this? 

Since this chart can also show stacked area charts, maybe getting data from all datasets would be complicated for other tooltips?

Let's exclude the line chart from this PR and analyse the difference.